crates/cli/Cargo.toml

```diff
@@ -11,7 +11,9 @@ name = "javy"
 path = "src/main.rs"
 
 [features]
+default = ["plugin-internal"]
 dump_wat = ["dep:wasmprinter"]
+plugin-internal = []
```

Comment:
I don't want to introduce the feature gates in this PR but rather when the codegen crate is created. That said, it would be helpful to mark off with PR comments as to which methods would be feature gated.